### PR TITLE
v4 - Added  mixin to handle media queries that span multiple breakpoints

### PR DESCRIPTION
**Pull request by dmbaughman**
Per conversation with @mdo RE: adding a media query mixin that spans multiple breakpoints.

```
// SCSS input
@include media-breakpoint-between(sm,md) {
  p {
    color: #00c;
  }
}

// CSS output
@media (min-width: 34em) and (max-width: 61.9em) {
  p {
    color: #00c; 
  }
}
```

### CODE REVIEW

**Comment by dmbaughman** [comment]
Fix for html linting error in separate PR: https://github.com/twbs/bootstrap/pull/17319

File: scss/mixins/_breakpoints.scss [hunk 1]
```diff
@@ -74,3 +74,13 @@
...
+
+// Media that spans multiple breakpoint widths.
+// Makes the @content apply between the min and max breakpoints
+@mixin media-breakpoint-between($lower, $upper, $breakpoints: $grid-breakpoints) {
+  @include media-breakpoint-up($lower, $breakpoints) {
+    @include media-breakpoint-down($upper, $breakpoints) {
+      @content;
+    }
+  }
+}
```
Comment (mdo) [inline scss/mixins/_breakpoints.scss]: Missing a new line here.

Comment (dmbaughman) [inline scss/mixins/_breakpoints.scss]: Fixed.

Comment (glebm) [inline scss/mixins/_breakpoints.scss]: Perhaps media-breakpoint-only should now just call media-breakpoint-between?

**Comment by mdo** [comment]
Can't recall the context around this PR... do we need this for something in Bootstrap's CSS, or just as a nicety?

**Comment by cvrebert** [comment]
@mdo It's a good and useful nicety.

**Comment by cvrebert** [comment]
Fixes #18014.

**Comment by mdo** [comment]
<3

**Pull request merged**